components/accordion/index.css

```diff
@@ -11,17 +11,295 @@ governing permissions and limitations under the License.
 */
 
 /* https://git.corp.adobe.com/Spectrum/spectrum-origins/pull/60 */
+
 .spectrum-Accordion {
-	/* Subtract the size of the border since it's on the item itself */
-	--spectrum-accordion-item-height-actual: calc(
-		var(--spectrum-accordion-item-height) -
-			var(--spectrum-accordion-item-border-size)
+	--spectrum-accordion-item-height: var(--spectrum-component-height-200);
+	--spectrum-accordion-item-width: var(--spectrum-accordion-minimum-width);
+	--spectrum-accordion-disclosure-indicator-height: var(
+		--spectrum-component-height-100
+	);
+	--spectrum-accordion-disclosure-indicator-to-text-space: var(
+		--spectrum-accordion-disclosure-indicator-to-text
+	);
+	/*--spectrum-accordion-edge-to-disclosure-indicator-space: var(--spectrum-accordion-edge-to-disclosure-indicator);*/
+	/*--spectrum-accordion-edge-to-text-space: var(--spectrum-accordion-edge-to-text);*/
```

Comment:
Can we make use of these tokens now? 
The `--spectrum-accordion-edge-to-disclosure-indicator` can be added to existing calculations; I think that would be padding-inline-start on the button, and inset-inline-start of the indicator. It can be tested with its accompanying --mod.

The `--spectrum-accordion-edge-to-text` should be used for padding-inline-end. Currently there is a browser default padding-right appearing on the button.